### PR TITLE
Adding Troubleshooting Workbook for Azure Blockchain Workbench

### PR DESCRIPTION
**Pull request by keikumata**

### CODE REVIEW

**Comment by gardnerjr** [comment]
@ericc1103 @acearun  can one of you verify the category/etc stuff for this PR?  the workbook itself seems fine.

File: Workbooks/Azure Blockchain/Workbench Troubleshooting/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Workbench Troubleshooting",
+    "author": "Azure Blockchain",
+    "order": 100,
```
Comment (acearun) [inline Workbooks/Azure Blockchain/Workbench Troubleshooting/settings.json]: You probably don't need this anymore. @ericc1103, can you confirm?

Comment (ericc1103) [reply Workbooks/Azure Blockchain/Workbench Troubleshooting/settings.json]: Right, we don't need it anymore since we already specify it within galleries.

Comment (keikumata) [reply Workbooks/Azure Blockchain/Workbench Troubleshooting/settings.json]: sounds good - I'll remove it

**Pull request merged**